plugins/postcss-media-minmax/test/calc.css

```diff
@@ -109,3 +109,41 @@
 @media (calc(10px / 3) > width >= calc(100px / 3)) {}
 
 @media (calc(10px / 3) >= width >= calc(100px / 3)) {}
+
+/* color */
+/* px */
+@media (color < sin(90deg)) {}
```

Comment:
`sin(90deg)` == `1`

_In the results only some outputs have `sin(90deg)` and others have a numeric value. Only when there is a need to solve the math expression will it do so, if the value can be used as is it will just preserve it._